requirements.in

```diff
@@ -1,25 +1,25 @@
-pycparser==2.14 # TEMPORARY
```

Comment:
Pycparser doesn't seem to need to be pinned. Any reason why we should keep this?